modules/common/src/main/mon.scala

```diff
@@ -159,8 +159,13 @@ object mon {
       object trace {
         def create = makeTrace("round.move.trace")
       }
-      val networkLag = rec("round.move.network_lag")
-      val lagLowEstimate = rec("round.move.lag_low_estimate")
+      val networkLag = rec("round.move.network_lag") // Deprecated.
```

Comment:
why do we need to keep it around?

Reply:
Only if we want graphana to have more history for client_lag before switching.